doc/version_history.rst

```diff
@@ -6,6 +6,19 @@
 Version History
 ###############
 
+v1.8.0
+------
+
+Changes:
+
+  * Update configuration schema to remove schema file and use module instead.
```

Comment:
The configs are all backwards compatible, yes? No new version is required?

Reply:
correct